reconcile/openshift_saas_deploy_wrapper.py

```diff
@@ -29,6 +35,11 @@ def run(dry_run, thread_pool_size=10, io_dir='throughput/'):
             thread_pool_size,
             len(saas_file_owners_diffs))
 
-    threaded.run(osd_run_wrapper, saas_file_owners_diffs, thread_pool_size,
-                 dry_run=dry_run,
-                 available_thread_pool_size=available_thread_pool_size)
+    exit_codes = threaded.run(
```

Comment:
but will this return at all? won't it just stay hanging here as long as there are running threads?

Reply:
catching `SystemExit` will make the thread return